libs/docs/platform/table/examples/platform-table-p13-columns-example.component.ts

```diff
@@ -29,6 +29,10 @@ export class PlatformTableP13ColumnsExampleComponent {
     constructor() {
         this.source = new TableDataSource(new TableDataProviderExample());
     }
+
```

Comment:
console log and unused method.

Reply:
Console logs in the example code are OK to demonstrate this type of thing, in my opinion

Reply:
ah sorry, i reviewed it like ordinary component, i will resolve this comment with myself.